pennylane/qchem/observable_hf.py

```diff
@@ -122,58 +122,60 @@ def qubit_observable(o_ferm, cutoff=1.0e-12):
     return o_qubit
 
 
-def jordan_wigner(op):
+def jordan_wigner(op, notation="physicist"):
     r"""Convert a fermionic operator to a qubit operator using the Jordan-Wigner mapping.
 
     For instance, the one-body fermionic operator :math:`a_2^\dagger a_0` should be constructed as
-    [2, 0] and the two-body operator :math:`a_4^\dagger a_3^\dagger a_2 a_1` should be constructed
-    as [4, 3, 2, 1].
+    [2, 0]. The two-body operator :math:`a_4^\dagger a_3^\dagger a_2 a_1` should be constructed
+    as [4, 3, 2, 1] with ``notation='physicist'``. If ``notation`` is set to ``'chemist'``, the
+    two-body operator [4, 3, 2, 1] is constructed as :math:`a_4^\dagger a_3 a_2^\dagger a_1`.
 
     Args:
         op (list[int]): the fermionic operator
+        notation (str): notation specifying the order of the two-body fermionic operators
 
-    Returns
-        tuple(list[complex], list[list[int, str]]): list of coefficients and the qubit-operator terms
+    Returns:
+        tuple(list[complex], list[Operation]): list of coefficients and qubit operators
 
     **Example**
 
     >>> f  = [0, 0]
     >>> q = jordan_wigner(f)
-    >>> q
-    ([(0.5+0j), (-0.5+0j)], [Identity(wires=[0]), PauliZ(wires=[0])]) # corresponds to :math:`\frac{1}{2}(I_0 - Z_0)`
+    >>> q # corresponds to :math:`\frac{1}{2}(I_0 - Z_0)`
+    ([(0.5+0j), (-0.5+0j)], [Identity(wires=[0]), PauliZ(wires=[0])])
     """
     if len(op) == 1:
-        op = [((op[0], 1),)]
+        op = [(op[0], 1)]
 
     if len(op) == 2:
-        op = [((op[0], 1), (op[1], 0))]
+        op = [(op[0], 1), (op[1], 0)]
 
     if len(op) == 4:
-        op = [((op[0], 1), (op[1], 1), (op[2], 0), (op[3], 0))]
-
-        if op[0][0][0] == op[0][1][0] or op[0][2][0] == op[0][3][0]:
-            return 0
-
-    for t in op:
-        for l in t:
-            z = [(index, "Z") for index in range(l[0])]
-            x = z + [(l[0], "X"), 0.5]
-
-            if l[1]:
-                y = z + [(l[0], "Y"), -0.5j]
-
-            else:
-                y = z + [(l[0], "Y"), 0.5j]
+        if notation == "physicist":
```

Comment:
Should we raise an error when the notation is neither `physicist` nor `chemist` when we have two-body terms?

Reply:
Added.